lizmap/modules/admin/templates/project_list_zone.tpl

```diff
@@ -107,7 +107,7 @@
             <td>{if !empty(trim($p['qgis_log']))}🔴{/if}</td>
 
             <!-- Hidden QGIS logs -->
-            <td>{$p['qgis_log']|nl2br}</td>
+            <td>{$p['qgis_log']|nl2br|strip_tags|eschtml:ENT_QUOTES}</td>
```

Comment:
won't strip_tags remove br tags generated by nl2br ?
```suggestion
            <td>{$p['qgis_log']|strip_tags|eschtml:ENT_QUOTES|nl2br}</td>
```

Reply:
yes, it will.